internal/core/items/text.rs

```diff
@@ -209,6 +172,237 @@ impl Text {
             italic: self.font_italic(),
         }
     }
+
+    fn color(self: Pin<&Self>) -> Brush {
+        self.color()
+    }
+
+    fn alignment(
+        self: Pin<&Self>,
+    ) -> (super::TextHorizontalAlignment, super::TextVerticalAlignment) {
+        (self.horizontal_alignment(), self.vertical_alignment())
+    }
+
+    fn wrap(self: Pin<&Self>) -> TextWrap {
+        self.wrap()
+    }
+
+    fn overflow(self: Pin<&Self>) -> TextOverflow {
+        self.overflow()
+    }
+
+    fn letter_spacing(self: Pin<&Self>) -> LogicalLength {
+        self.letter_spacing()
+    }
+
+    fn stroke(self: Pin<&Self>) -> (Brush, LogicalLength, TextStrokeStyle) {
+        (self.stroke(), self.stroke_width(), self.stroke_style())
+    }
+}
+
+/// The implementation of the `Text` element
+#[repr(C)]
+#[derive(FieldOffsets, Default, SlintElement)]
+#[pin]
+pub struct SimpleText {
+    pub width: Property<LogicalLength>,
+    pub height: Property<LogicalLength>,
+    pub text: Property<SharedString>,
+    pub font_size: Property<LogicalLength>,
+    pub font_weight: Property<i32>,
+    pub color: Property<Brush>,
+    pub horizontal_alignment: Property<TextHorizontalAlignment>,
+    pub vertical_alignment: Property<TextVerticalAlignment>,
```

Comment:
I think I would've left those two out :)

Reply:
I looked at the usage in czkawka/krokiet and many Text have this property defined